taccsite_cms/templates/header_logo.html

```diff
@@ -5,7 +5,7 @@
   {% with filename=logo|index:1 selectors=logo|index:2 targeturl=logo|index:3 targettype=logo|index:4 accessibility=logo|index:5 corstype=logo|index:6 visibility=logo|index:7 %}
     {% if visibility == "True" %}
     <a id="header-logo" class="navbar-brand {{className}}" href="{{ targeturl }}" target="{{ targettype }}">
-      <img class="portal-logo {{ selectors }}" src="{% static filename %}" crossorigin="{{ corstype }}" alt="{{ accessibility }}" >
+      <img class="portal-logo {{ selectors }}" src="{% if "://" in filename %}{{ filename }}{% else %}{% static filename %}{% endif %}" crossorigin="{{ corstype }}" alt="{{ accessibility }}" />
```

Comment:
Would the new `is_remote` param be possible to key off here for static vs non-static?

Reply:
**Yes, with effort.** The FAVICON is a dict, so adding `is_remote` was easy. The LOGO is a list, so adding `is_remote` could be cumbersome.

### Ideas

**A.** Add _optional_ entry to LOGO setting. Add _careful_ logic to avoid "index out of range" error in template. Test _thoroughly_.
**B.** Add new TACC_LOGO dict setting. Deprecate LOGO list setting. Have template accept both, until all sites are updated.
**C.** Add LOGO_IS_REMOTE setting. Maintain both.

I prefer **B**.

Reply:
Though cumbersome now to refactor, I agree B sounds best for longevity

Reply:
I committed changes to support B:
- `TACC_LOGO` and `TACC_FAVICON` are standard[^1]
- `LOGO` and `FAVICON` take precedence, for backwards-compatibility

[^1]: Other. newer Core-CMS–specific settings are prefixed with `TACC_`, to mirror `DJANGO_` and `CMS_` and `BLOG_`.

I tested, but haven't taken screenshots yet.

Reply:
Okay. UI for testing on local Core-CMS. Re-deployed pprd PTDataX.

Reply:
You may have addressed this elsewhere, but curious on your thoughts of using `PORTAL_` as a prefix instead of `TACC_`

Reply:
My thoughts are _okay_… I've noticed the problem with `TACC_`, but `TACC_` is consistent... I will try to write a Python function to cleanly support `TACC_` and `PORTAL_` until `TACC_` is deprecated. Or, I give up and just do it for these. I want to give more attention on this. I will return.

Reply:
I'm totally fine with merge now, change later in this case if more convenient.

Reply:
I put some thought into it.

1. **We CAN support new prefix LATER**,
    - via some slim Python code I don't know how to write yet.
2. **I think prefix should not be `PORTAL_`**, because:
    - clarity of Core-Portal vs Core-CMS settings seems important
    - it propagates ill-defined word "Portal"[^1]
3. **The prefix should NOT be `CMS_`**, because:
    - Django CMS uses that prefix
4. **I am tempted to use `CORE_CMS_`**, because:
    - the `CMS_` shows Core-CMS and Django CMS relationship
    - the `CORE_` shows Core-CMS and Core-Portal relationship


[^1]: TACC's website-portal-projects are, each as a whole, called "portals" even though they are a combination of a CMS **website** — the "(usually) not behind the login" aspect — and a Portal **app** —the "(mostly) behind the login" aspect.

Reply:
[CMD-79: Prefix CMS Settings w/ `CORE_CMS_` not `TACC_`](https://tacc-main.atlassian.net/browse/CMD-79)

And, **ugh**, I will not use slim Python code for dynamic variables to support both prefixes. I will use a manual list of aliases, for legibility. Chat GPT made a valid point when I asked it for help... AI is not useless in programming (just limited).

Reply:
I would say we are a bit beyond point 2, as in we are invested in that prefix/terminology. I'm not against migrating references of PORTAL to CORE_PORTAL across the board though, but that would be a separate task in my view.

Reply:
Yeah, I spent last night and this morning trying not to submit to a `PORTAL_`. **I submit!**[^1] I've renamed the [CMD-79](https://tacc-main.atlassian.net/browse/CMD-79) ticket, and will use `PORTAL_`.

[^1]: From one perspective, it is all a portal. The CMS is a small piece. From a different perspective, CMS surrounds the webpages and the portal. From yet another perspective the CMS and Portal are both optional pieces of a whole… that is still called Portal.

Reply:
- #792